### PR TITLE
Update gNMI sections

### PR DESCRIPTION
**Pull request by seiferteric**

### CODE REVIEW

File: doc/mgmt/Management Framework.md [hunk 1]
```diff
@@ -566,12 +566,12 @@ Client applications can use swagger generated client SDK or any other REST clien
 
 ##### 3.2.2.3 gNMI Client
 
-SONiC Teleletry service provides the gNMI server, while the client must be provided by the user.
+SONiC Telemetry service provides the gNMI server, while the client must be provided by the user. gNMI is typically used as a programmatic interface and therefor it is typically called directly from programming language environments using their respective gRPC libraries (https://github.com/grpc/grpc). For testing and scripting purposes, several CLI programs are provided as well.
```
Comment (jeffyin-dell) [inline doc/mgmt/Management Framework.md]: Change "therefor" to "therefore"

File: doc/mgmt/Management Framework.md [hunk 2]
```diff
@@ -566,12 +566,12 @@ Client applications can use swagger generated client SDK or any other REST clien
...
 
-GNMI client developed by JipanYANG.(github.com/jipanYANG/gnxi/gnmi_get, github.com/jipanYANG/gnxi/gnmi_set)
-is used for testing. gnmi_get and gnmi_set code has been changed to handle module name.
+GNMI clients developed by google and modified by JipanYANG.(github.com/jipanYANG/gnxi/gnmi_get, github.com/jipanYANG/gnxi/gnmi_set)
+was taking and further modified to support new features. gnmi_cli from openconfig (https://github.com/openconfig/gnmi/tree/master/cmd/gnmi_cli) is also taken for testing subscribe and capabilities operations. Finally, and new client was developed gnoi_client, for test gNOI RPC operations.
```
Comment (jeffyin-dell) [inline doc/mgmt/Management Framework.md]: "taking" to "taken" ?

Comment (jeffyin-dell) [inline doc/mgmt/Management Framework.md]: "a new client" instead of "and new client"

File: doc/mgmt/Management Framework.md [hunk 3]
```diff
@@ -580,12 +580,13 @@ Supported RPC Operations:
     + Update: List of one or more objects to update
     + Replace: List of one or objects to replace existing objects, any unspecified fields wil be defaulted.
     + Delete: List of one or more object paths to delete
-- Capabilities: Return gNMI version and list of supported models
+- Capabilities: Return gNMI version and list of supported models and model versions.
+    + An gNMI Extension field has been added as well to return the SONiC model "bundle version".
```
Comment (jeffyin-dell) [inline doc/mgmt/Management Framework.md]: "An" to "A"

File: doc/mgmt/Management Framework.md [hunk 4]
```diff
@@ -580,12 +580,13 @@ Supported RPC Operations:
...
 - Subscribe:
     + Subscribe to paths using either streaming or poll, or once based subscription, with either full current state or updated values only.
         * Once: Get single subscription message.
         * Poll: Get one subscription message for each poll request from the client.
-        * Stream: Get one subscription message for each object update, or at each sample interval if using sample mode. target_defined uses the values pre-configured for that particular object.
+        * Stream: Get one subscription message for each object update (called ON_CHANGE mode), or at each sample interval if using sample mode. target_defined uses the values pre-configured for that particular object. Not all paths support ON_CHANGE mode due to performance considerations, while all paths will support sample mode and therfore target defined as well since it will default to sample if ON_CHANGE is not supported.
```
Comment (jeffyin-dell) [inline doc/mgmt/Management Framework.md]: "therefore" spelled incorrectly?

File: doc/mgmt/Management Framework.md (hunk 1 above)
Comment (seiferteric) [reply doc/mgmt/Management Framework.md]: fixed

File: doc/mgmt/Management Framework.md (hunk 2 above)
Comment (seiferteric) [reply doc/mgmt/Management Framework.md]: fixed

File: doc/mgmt/Management Framework.md (hunk 4 above)
Comment (seiferteric) [reply doc/mgmt/Management Framework.md]: fixed

File: doc/mgmt/Management Framework.md (hunk 3 above)
Comment (seiferteric) [reply doc/mgmt/Management Framework.md]: fixed

File: doc/mgmt/Management Framework.md (hunk 2 above)
Comment (seiferteric) [reply doc/mgmt/Management Framework.md]: fixed

**Review by jeffyin-dell — APPROVED** [review]

**Pull request merged**